CoreHelp/src/au/gov/asd/tac/constellation/help/ConstellationHelpDisplayer.java

```diff
@@ -240,11 +240,14 @@ public boolean display(final HelpCtx helpCtx) {
 
         final String helpAddress = HelpMapper.getHelpAddress(helpId);
         // use the requested help file, or the About Constellation page if one is not given
-        final String helpLink = StringUtils.isNotEmpty(helpAddress) ? helpAddress.substring(2) : helpDefaultPath;
+        String helpLink = StringUtils.isNotEmpty(helpAddress) ? helpAddress.substring(2) : helpDefaultPath;
 
         try {
             final String url;
             if (isOnline) {
+                if (helpLink.contains("constellation-adaptors") || helpLink.contains("constellation-cyber")) {
```

Comment:
while acknowledging that the online help will only additionally add page from these repos, I wonder whether it would be good to have a more generic check that covers both of these without explicitly calling them out (if possible)